lib/pages/sign_up/widgets/signup_form.dart

```diff
@@ -33,12 +54,19 @@ class SignUpFormState extends State<SignUpForm> {
               height: 15.0,
             ),
             new TextFormField(
-              decoration: InputDecoration(labelText: 'Enter password'),
+              controller: _passwordController,
+              decoration: InputDecoration(
+                labelText: 'Enter password',
+                border: new OutlineInputBorder(
+                    borderSide: new BorderSide(color: Colors.grey)),
+              ),
               obscureText: true,
               keyboardType: TextInputType.text,
               validator: (value) {
                 if (value.isEmpty) {
```

Comment:
Move form validators to one place (Like FormValidators class)

Reply:
For this feature needs one more issue

Reply:
Fix in #22 22